tests/unit/s2n_certificate_test.c

```diff
@@ -804,5 +804,41 @@ int main(int argc, char **argv)
         EXPECT_BYTEARRAY_EQUAL(chain->sct_list.data, sct_list, sizeof(sct_list));
     };
 
+    /* Test s2n_cert_chain_and_key_load_cns */
+    {
+        DEFER_CLEANUP(struct s2n_cert_chain_and_key *chain = NULL,
+                s2n_cert_chain_and_key_ptr_free);
+        EXPECT_SUCCESS(s2n_test_cert_chain_and_key_new(&chain,
+                S2N_RSA_2048_SHA256_INTERMEDIATE_CERT_CUSTOM_OID,
+                S2N_RSA_2048_SHA256_INTERMEDIATE_CA_KEY));
```

Comment:
Is there any particular reason you chose these certs specifically?

Reply:
I just copied it from https://github.com/aws/s2n-tls/blob/main/tests/unit/s2n_certificate_test.c#L404C6-L406